_dashboard/DotNetPerfMonitor/composables/use_benchmark_grouper.ts

```diff
@@ -0,0 +1,16 @@
+export default function useBenchmarkGrouper(
+  data: Array<{ field: String; [key: string]: any }>,
+  benchmark: String
+) {
+  return data.filter((item) => item.solution === benchmark);
+}
+
+const solutions = [
+  "LargeAppCPM-142722b",
+  "LargeAppCPM64-142722b",
+  "LargeAppCPM-nostaticgraph-142722b",
+  "LargeAppCPM64-nostaticgraph-142722b",
+  "Orleans-eda972a",
+  "OrchardCore-5dbd92c",
+  "NuGetClient-win-d76a117",
+];
```

Comment:
This should be dynamic so we can add easily new benchmarks without updating the frontend code.